src/app/courses/courses.scss

```diff
@@ -0,0 +1,50 @@
+@import '../variables';
```

Comment:
You've just copied the exact style definitions from `resources.scss`.  There are some style definitions missing from the previous version for CoursesComponent, some definitions that are irrelevant, and some which make the course view worse.  

Please go back to the previous version and only add style definitions required for this PR.